01week/rockPaperScissors.js

```diff
@@ -8,10 +8,25 @@ const rl = readline.createInterface({
 });
 
 
-function rockPaperScissors(hand1, hand2) {
-
-  // Write code here
+function rockPaperScissors(cleanHand1, cleanHand2){
+  let hand1 = cleanHand1.trim().toLowerCase();
+  let hand2 = cleanHand2.trim().toLowerCase();
 
+  if (hand1 == 'rock' && hand2 == 'scissors'){
+    return "Hand one wins!";
+  } else if (hand1 == 'rock' && hand2 == 'paper'){
+    return "Hand two wins!";
+  } else if (hand1 == 'paper' && hand2 == 'rock'){
+    return "Hand one wins!";
+  } else if (hand1 == 'paper' && hand2 == 'scissors'){
+    return "Hand two wins!";
+  } else if (hand1 == 'scissors' && hand2 == 'paper'){
+    return "Hand one wins!";
+  } else if (hand1 == 'scissors' && hand2 == 'rock'){
+    return "Hand two wins!";
+  } else if (hand1 == hand2){
+    return "It's a tie!";
+  }
 }
 
```

Comment:
@rmcbride2121 
The assignment was to add these tests: https://javascript.austincodingacademy.com/_book/02Week/02DayClass.html#instructions
I can't give you credit for this unless you add them, thanks!

Reply:
Hey Anthony! I resubmitted this project